core/src/test/scala/org/broadinstitute/dsde/rawls/entities/datarepo/DataRepoBigQuerySanitizationSpec.scala

```diff
@@ -0,0 +1,70 @@
+package org.broadinstitute.dsde.rawls.entities.datarepo
+
+import java.nio.charset.Charset
+
+import com.google.common.io.Resources
+import org.scalatest.{BeforeAndAfterAll, FreeSpec}
+import spray.json._
+
+class DataRepoBigQuerySanitizationSpec extends FreeSpec with DataRepoBigQuerySupport with BeforeAndAfterAll {
+
+  // read the list of naughty strings from https://github.com/minimaxir/big-list-of-naughty-strings.
+  // We expect a json array, containing base64-encoded strings.
+  val blnsData = Resources.toString(
+    new java.net.URL("https://raw.githubusercontent.com/minimaxir/big-list-of-naughty-strings/master/blns.base64.json"),
+    Charset.defaultCharset())
+
+  val naughtyStrings = blnsData.parseJson match {
+    case blns:JsArray =>
+      blns.elements.map {
+        case s:JsString => new String(java.util.Base64.getDecoder.decode(s.value))
+        case _ => fail("found something not a string in BLNS")
+      }.toList
+    case _ => fail("BLNS was not a json array")
+  }
+
+  override protected def beforeAll(): Unit = {
+    // create a new variable to hold the size; this prevents Scalatest from attempting to output
+    // anything about the contents of the list itself, if the assertion fails
+    val nSize = naughtyStrings.size
+    assert(nSize > 0, "fixture data was empty; failing test. Size: " + nSize)
+
+    super.beforeAll()
+  }
```

Comment:
I create the `naughtyStrings` val at the class level, so that when Scalatest calculates its test names, the dynamic string `s"do the right things across ${naughtyStrings.size} ...` has the correct value. But I validate via a `beforeAll` that we successfully downloaded and parsed the data.